src/cisco-decrypt.c

```diff
@@ -1,21 +1,12 @@
-/* Decoder for password encoding of Cisco VPN client.
-   Copyright (C) 2005 Maurice Massar
+/*
+  Decoder for password encoding of Cisco VPN client.
 
-   Thanks to HAL-9000@evilscientists.de for decoding and posting the algorithm!
+  SPDX-FileCopyrightText: 2023 Jolla Ltd.
+  SPDX-FileCopyrightText: 2005 Maurice Massar
```

Comment:
Please use a consistent sort method throughout the codebase: in some files you sorting years ascendingly, in some others descendingly.

Reply:
@Thaodan this still applies for the last push: you're not using the same consistent sorting method for `SPDX-FileCopyrightText` header. Please, descending sort all the entries (e.g. in `src/decrypt-utils.c` the order is ascending, or in `src/dh.c` is not ordered at all). Once that is fixed, I'm going to merge the PR, thanks!

Reply:
@streambinder Oh sorry :sweat_smile: . I sorted them all now in descending order from oldest to newest.